### ISSUE TITLE
Prevent Gel Bomb spamming

### ISSUE THREAD

**Issue by Aneonen** [BEEmod/BEE2-items#4304]
### Description of the feature

A change to the gel bomb dropper that prevents gel bombs from being spammed out of the bomb dropper by repeatedly firing an input.

### Why should this be implemented

Right now, if a gel bomb dropper's input is held (e.g. by a floor button), a single bomb will drop, and another will only appear when the first bomb explodes. However, if the player steps off of and back onto the button while the first gel bomb is still alive, a second will drop. This new bomb can then be abused by the player, or can collide with the first to break it by accident, etc. If the player spams the input (for a general input), they can spam gel bombs. 
By forcing the dropper to only be able to drop a new bomb once the previous one has been destroyed, this behavior is corrected and gel bomb droppers are made much more consistent and workable.

Whether this is a "bug" to be fixed or an extra feature to be added depends on the perception of how the gel dropper should work. I personally think the rule of allowing only one bomb to be out at a time is generative and leads to interesting bomb-preservation puzzles, but I can see why some players might intuit that the bombs should be able to be dropped repeatedly. In any case, it seems like the dropper occupies an awkward middle ground right now; bombs don't drop continuously like the other gel streams, but they don't seem to observe the rule articulated by an automatic, disconnected bomb dropper.

### Additional information

I personally edited my bomb dropper to use a coop manager (instead of a relay) to make sure a bomb would only drop if the input was active and the current bomb had popped.

**Comment by HeimlichLaboratories** [comment]
This is a very good idea! It should be implemented.

**Comment by TeamSpen210** [comment]
Indeed, there's a number of different ways we could have the logic setup for bomb gel droppers. In addition to these two, there's also the option of having it just drop repeatedly (probably with a limit configured).

**Comment by HeimlichLaboratories** [comment]
I mean, that's how they worked in the campaign, right? Would be useful
